eZ/Publish/SPI/Persistence/Content/Handler.php

```diff
@@ -170,6 +170,8 @@ public function deleteVersion($contentId, $versionNo);
     /**
      * Returns the versions for $contentId.
      *
+     * Result is returned with oldest version first (sorted by created, alternatively version id if auto increment).
```

Comment:
Unless I'm missing something, is should be possible define this as sorted by version number, this is `VersionInfo::$versionNo`, `ezcontentobject_version.version` in Legacy DB.